IntroToCSharp/Pages/Lessons/Collections/Arrays.razor

```diff
@@ -0,0 +1,379 @@
+@inherits CaptainCoder.Page
+@page "/lessons/collections/arrays"
+
+<Title>Arrays</Title>
+
+
+<Section Title="Introdution">
+    <MudText>
```

Comment:
MudText's don't have good spacing for writing paragraphs so I created a new "<Text>" component that you should use instead.